test_reporting/junit_xml_parser.py

```diff
@@ -209,8 +210,7 @@ def validate_junit_xml_archive(directory_name, strict=False):
             print(f"could not parse {document}: {e} - skipping")
 
     if not roots:
-        raise JUnitXMLValidationError(f"provided directory {directory_name} does not contain any valid XML files")
-
+        print("provided directory {} does not contain any valid XML files".format(directory_name))
```

Comment:
Why not to raise exception here?

Reply:
@wangxin If file doesn't exist, we should continue the test report uploading process to upload pipeline results and summary. Don't expect the script failed anymore.